lib/texts.js

```diff
@@ -127,5 +127,5 @@ ${leaves}`,
 
 	// Cli args
 	cliLogLevel: () => 'Logger verbosity level',
-	cliUseSentry: () => 'Allow suitest to use Sentry error reporting',
+	cliDisallowCrashReports: () => 'Allow suitest to use Sentry error reporting',
```

Comment:
Please, replace error message with "Disallow Suitest to submit automatic crash reports when unexpected exceptions occur"